main/postgres/initdb.d/README.markdown

```diff
@@ -0,0 +1,14 @@
+This directory helps you prepare your Postgres instance for MPortal to run.
+
+Context: when MPortal runs for the first time, it will create all the Postgres tables it needs;
+but to do that, it assumes that certain roles/"schemas" are already set up in Postgres.
+
+Here are instructions for how to take a fresh Postgres instance and make it ready for MPortal to run for the first time:
+
+- First, create two users (probably by logging in as the root user via PGAdmin), named `metrics_dba` and `akka_dba`.
+- `export POSTGRES_USER=postgres` (or whatever your root user's name is)
+- `export POSTGRES_DB=????` (TODO(spencerpearson): what is this? Our production Postgres seems to have databases named `postgres` and `rdsadmin`, plus `metrics` which these scripts seem to create.)
+- `export POSTGRES_PASSWORD=<your secret postgres-root-user password>`
+- locally `chmod 700 ./init.sh` and edit the `export PGPASSWORD="..."` lines to contain the actual passwords for the metrics_dba and akka_dba users
+- `./init.sh`
```

Comment:
I ran the SQL by hand when I setup the production database so that I could see the results of each step (and considering the cost of failure). I don't know what real DBAs prefer to do, but personally I would not point a script like this at a production database instance!